### ISSUE TITLE
[Bug Report] not able to capture checked or unchecked in el-checkbox-group

### ISSUE THREAD

**Issue by amitneg1** [ElemeFE/element#5865]
### Element UI version
1.3.7

### OS/Browsers version
in all

### Vue version
2.4.1

### Reproduction Link
[https://github.com/Monotype/cs-mep-ui](https://github.com/Monotype/cs-mep-ui)

### Steps to reproduce
Here is my code 
<el-checkbox-group
                        fill="#2A9AB7"
                        size="small"
                        text-color="#ffffff"
                        v-model="campaign.addons_services"
                        @change="handleChange">
                        <el-checkbox-button
                          v-for="service in optionsAddonsServices"
                          :key="service.value"
                          :label="service.value"
                          :title="service.description"
                          :disabled="service.lockedIn">
                          {{ service.label }}
                        </el-checkbox-button>
                      </el-checkbox-group>

Step1 : In checkbox group change event not able to get whether el-checkbox-button is checkout or not.

### What is Expected?
Here is my code 
<el-checkbox-group
                        fill="#2A9AB7"
                        size="small"
                        text-color="#ffffff"
                        v-model="campaign.addons_services"
                        @change="handleChange">
                        <el-checkbox-button
                          v-for="service in optionsAddonsServices"
                          :key="service.value"
                          :label="service.value"
                          :title="service.description"
                          :disabled="service.lockedIn">
                          {{ service.label }}
                        </el-checkbox-button>
                      </el-checkbox-group>

Expected is to get value of checked or unchecked el-checkbox-button in @change handler function

### What is actually happening?
Not able to get checked or unchecked value in @change handler function

**Comment by Leopoldthecoder** [comment]
I'm not sure if I understand your problem: do you wish to know if one more button or one less button is checked in the `change` event handler?

**Comment by amitneg1** [comment]
@Leopoldthecoder Thanks for your quick response. I have got an answer of my question at my end.
Let me rephrase my question once again. My question was "how to get checked/unchecked value  of el-checkbox-button component".

**Comment by Leopoldthecoder** [comment]
https://jsfiddle.net/ja3na86g/1/ I've written a demo so that our discussion has something to base on. Could you tell me what do you expect to be printed in this demo?

**Comment by amitneg1** [comment]
First of all, Thanks for your effort. 
Here in this handler function can i get whether button is checked or unchecked.
handleChange(val) {
console.log(val);
}

**Comment by Leopoldthecoder** [comment]
Honestly I don't know what 'whether button is checked or unchecked' means. That's why I wrote that demo.

In the `handleChange` handler, `val` is an array of all checked values. For example, run the above demo and click on button `1`, you'll see `['3', '1']` in the console. Could you tell me what do you expect to see in the console when you click button `1`?

**Comment by amitneg1** [comment]
I want true/false to be console, when button 1 checked/unchecked.

**Comment by Leopoldthecoder** [comment]
So when the `change` event fires, you want to know which button is clicked and if it is checked or unchecked. Actually this can be done on your end, e.g. diff the previous checked array and the new array. So we probably will leave `change` event as it is.
